crates/superposition_types/migrations/2024-12-19-095525_create_organisation_table/up.sql

```diff
@@ -0,0 +1,25 @@
+-- up.sql
+CREATE SCHEMA IF NOT EXISTS superposition;
+
+CREATE TYPE superposition.org_status AS ENUM ('ACTIVE', 'INACTIVE', 'PENDINGKYB');
+
+CREATE TABLE IF NOT EXISTS superposition.organisation (
+    id VARCHAR(30) PRIMARY KEY NOT NULL, 
```

Comment:
we planned to keep the org id of fixed length 25 right ?
to allow for some buffer in future?